src/main/java/lotto/model/LottoReward.java

```diff
@@ -0,0 +1,48 @@
+package lotto.model;
+
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import lotto.LottoMatchResult;
+
+public enum LottoReward {
+    firstPlace(6, 2000000000),
```

Comment:
- 열거형 상수의 이름은 관례에 따라 대문자로 작성해보면 어떨까요?
- 언더스코어(`_`)를 활용하면 단위가 큰 숫자의 가독성을 향상시킬 수 있을 것 같아요.
```suggestion
    FIRST_PLACE(6, 2_000_000_000),
```